geomesa-accumulo/geomesa-accumulo-tools/src/test/scala/org/locationtech/geomesa/accumulo/tools/ingest/ShpIngestTest.scala

```diff
@@ -42,81 +43,84 @@ class ShpIngestTest extends Specification {
   val baseArgs = Array[String]("ingest", "--zookeepers", "zoo", "--mock", "--instance", "mycloud", "--user", "myuser",
     "--password", "mypassword", "--catalog", "testshpingestcatalog")
 
+  var dir: Path = _
+  var shpFile: File = _
+
   "ShpIngest" should {
 
     "should properly ingest a shapefile" in {
-      withShapeFile { file =>
-        val args = baseArgs :+ file.getAbsolutePath
+      val args = baseArgs :+ shpFile.getAbsolutePath
 
-        val command = AccumuloRunner.parseCommand(args).asInstanceOf[AccumuloDataStoreCommand]
-        command.execute()
+      val command = AccumuloRunner.parseCommand(args).asInstanceOf[AccumuloIngestCommand]
+      command.setConsole(new AnyRef {
+        def readLine(): String = "y" // accept prompt to use inferred schema
+        def readPassword(): Array[Char] = Array.empty
+      })
+      command.execute()
 
-        val fs = command.withDataStore(_.getFeatureSource("shpingest"))
+      val fs = command.withDataStore(_.getFeatureSource("shpingest"))
 
-        SelfClosingIterator(fs.getFeatures.features).toList must haveLength(2)
+      SelfClosingIterator(fs.getFeatures.features).toList must haveLength(2)
 
-        val bounds = fs.getBounds
-        bounds.getMinX mustEqual 10.0
-        bounds.getMaxX mustEqual 20.0
-        bounds.getMinY mustEqual 30.0
-        bounds.getMaxY mustEqual 40.0
+      val bounds = fs.getBounds
+      bounds.getMinX mustEqual 10.0
+      bounds.getMaxX mustEqual 20.0
+      bounds.getMinY mustEqual 30.0
+      bounds.getMaxY mustEqual 40.0
 
-        command.withDataStore { (ds) =>
-          ds.stats.getAttributeBounds[Date](ds.getSchema(schema.getTypeName), "dtg").map(_.tuple) must
-              beSome((features.head.getAttribute("dtg"), features.last.getAttribute("dtg"), 2L))
-        }
+      command.withDataStore { ds =>
+        ds.stats.getAttributeBounds[Date](ds.getSchema(schema.getTypeName), "dtg").map(_.tuple) must
+            beSome((features.head.getAttribute("dtg"), features.last.getAttribute("dtg"), 2L))
       }
     }
 
     "should support renaming the feature type" in {
-      withShapeFile { file =>
-        val args = baseArgs ++ Array("--feature-name", "changed", file.getAbsolutePath)
+      val args = baseArgs ++ Array("--feature-name", "changed", shpFile.getAbsolutePath)
 
-        val command = AccumuloRunner.parseCommand(args).asInstanceOf[AccumuloDataStoreCommand]
-        command.execute()
+      val command = AccumuloRunner.parseCommand(args).asInstanceOf[AccumuloIngestCommand]
+      command.setConsole(new AnyRef {
+        def readLine(): String = "y" // accept prompt to use inferred schema
```

Comment:
Are users required to enter 'y'?  I.e., use of the geomesa ingest command around shapefiles is changed by this PR, right?

Reply:
yeah, previously it would just ingest without prompting